piecrust-uplink/src/state.rs

```diff
@@ -134,10 +136,38 @@ where
     })
 }
 
+/// Calls a `contract`'s `fn_name` function with the given argument `fn_arg`.
+/// The contract will have `93%` of the remaining points available to spend.
+///
+/// To specify the points allowed to be spent by the called contract, use
+/// [`call_with_limit`].
 pub fn call<A, Ret>(
-    mod_id: ContractId,
-    name: &str,
-    arg: &A,
+    contract: ContractId,
+    fn_name: &str,
+    fn_arg: &A,
+) -> Result<Ret, ContractError>
+where
+    A: for<'a> Serialize<StandardBufSerializer<'a>>,
+    Ret: Archive,
+    Ret::Archived: Deserialize<Ret, Infallible>,
+{
+    call_with_limit(contract, fn_name, fn_arg, 0)
+}
+
+/// Calls a `contract`'s `fn_name` function with the given argument `fn_arg`,
+/// allowing it to spend the given `points_limit`.
+///
+/// A points limit of `0` is equivalent to using [`call`], and will use the
+/// default behavior - i.e. the called contract gets `93%` of the remaining
+/// points.
+///
+/// If the points limit given is not under the remaining amount, the default
```

Comment:
Maybe better say "If the points limit given is *above or equal* the remaining amount, ..."

Reply:
Positive is better than negative. Done.